include/clang/CrossTU/CrossTranslationUnit.h

```diff
@@ -147,6 +148,7 @@ class CrossTranslationUnitContext {
   bool isInvalidFunction(const FunctionDecl *FD);
 
 private:
+  void lazyInitLookupTable(TranslationUnitDecl *ToTU);
```

Comment:
@balazske This was the line with the `*` at wrong place. It is corrected now. It is my bad that I did an amend and then a force push, next time I'll create a separate commit even for the smallest changes to make review easier.